docs/mlapi-basics/networkvariable.md

```diff
@@ -11,7 +11,7 @@ There is a  Video Tutorial covering some of the concepts covered in this page [h
 
```

Comment:
again (picking up on this file arbitrarily), you might consider renaming `docs/mlapi-basics/` folder as `docs/netcode-basics` or `docs/basics`